sdks/java/extensions/euphoria/euphoria-core/src/main/java/org/apache/beam/sdk/extensions/euphoria/core/translate/coder/RegisterCoders.java

```diff
@@ -0,0 +1,149 @@
+package org.apache.beam.sdk.extensions.euphoria.core.translate.coder;
```

Comment:
We don't have license header check?

Reply:
My fault. Surprisingly enough the RAT (license check) is not invoked wen building `...euphoria-core` module. Should be fixed now.